src/messaging/dotnet/src/Client/Client/MessageRouterClient.cs

```diff
@@ -185,7 +185,7 @@ private Topic GetTopic(string topicName)
     }
 
     private string? _clientId;
-    private readonly IConnection _connection;
+    private IConnection _connection;
```

Comment:
Nitpick: why did you remove `readonly`?

Reply:
That was totally unintentional, thanks. I'll fix it in a sec.